src/vm/moar/dispatchers.nqp

```diff
@@ -2843,7 +2875,7 @@ nqp::register('raku-multi-core',
 
         # Obtain the candidate list, producing it if it doesn't already exist
         my $target := nqp::captureposarg($capture, 0);
-        my @candidates := $target.dispatch_order;
+        my @candidates := multi-filter-revision-gated-candidates($target);
```

Comment:
Same here: I'd like to see a quick test whether the target is even revision-gated before calling the helper.